docs/modules/commons/pages/expressions.adoc

```diff
@@ -419,3 +419,29 @@ When I initialize Scenario variable `listOfMaps` with values:
 |3  |
 Then `1-2-3` is = `#{evalGroovy(return listOfMaps.collect{it['key']}.sort().join('-'))}`
 ----
+
+== Null generation
+
+=== `null`
+
+Generates the `null`. Unlike the expression `eval(null)` generates not the string with `null` value, but the https://docs.oracle.com/javase/specs/jls/se11/html/jls-3.html#jls-3.10.7[special literal] of the null type.
+
+[source, subs="+quotes"]
+----
+#{null}
+----
+
+NOTE: Null generation expression doesn't work in examples tables.
+
+.Generation `null` and comparison with JSON null
+[source,gherkin]
+----
+When I initialize the scenario variable `json` with value `
+{
+    "persons": {
+        "nemo": null
+    }
+}
+`
+Then JSON element value from `${json}` by JSON path `$.persons.nemo` is equal to `#{null}`
+----
```

Comment:
can we add one more example for SQL?

Reply:
> can we add one more example for SQL?

Do we really need this?
I mean, if we have next code:
`When I execute SQL query `INSERT INTO test.launch_rockets (name, payload_cost, country) VALUES ('Long March', NULL, 'China');` against `testh2db-modifiable`
When I execute SQL query `SELECT payload_cost FROM test.launch_rockets WHERE country='China'` against `testh2db-modifiable` and save result to scenario variable `null-data``
Result data will be equals this:
`Then `${null-data}` is equal to `[{payload_cost=null}]``

Reply:
> can we add one more example for SQL?

> can we add one more example for SQL?

Do we really need this?
I mean, if we have next code:
`` When I execute SQL query `INSERT INTO test.launch_rockets (name, payload_cost, country) VALUES ('Long March', NULL, 'China');` against `testh2db-modifiable`
When I execute SQL query `SELECT payload_cost FROM test.launch_rockets WHERE country='China'` against `testh2db-modifiable` and save result to scenario variable `null-data` ``
Result data will be equals this:
``Then `${null-data}` is equal to `[{payload_cost=null}]` ``

Reply:
```gherkin
When I execute SQL query `SELECT * FROM test.launch_rockets` against `vividus-db` and save result to scenario variable `launch-rockets`
Then `${launch-rockets[0].payload_cost}` is equal to `#{null}`
```
I meant something like that

Reply:
we don't need to provide a full context, just a snippet